### PR TITLE
Makes Corgi collars removable

### PR DESCRIPTION
**Pull request by ShizCalev**
:cl: ShizCalev
tweak: Corgi collars can now be removed!
/:cl:

also added a missing canusetopic check and cleaned up a little bit.

### CODE REVIEW

**Comment by Iamgoofball** [comment]
did we make it so people can't wear them for their petplay fetishes yet

**Comment by Anonmare** [comment]
Yes Goof, you can put down the burning cross now

**Comment by QualityVan** [comment]
It was reverted though. Apparently.

**Comment by davethwave** [comment]
SHHH, that was supposed to keep a secret.

**Comment by Cw3040** [comment]
you weren't supposed to tell them that it was reverted you damned fool

File: code/modules/mob/living/simple_animal/friendly/dog.dm [hunk 1]
```diff
@@ -28,13 +28,30 @@
...
-	var/obj/item/inventory_head
-	var/obj/item/inventory_back
-	var/nofur = 0 		//Corgis that have risen past the material plane of existence.
 	gold_core_spawnable = FRIENDLY_SPAWN
 	can_be_held = TRUE
 	collar_type = "corgi"
+	var/obj/item/inventory_head
+	var/obj/item/inventory_back
+	var/shaved = 0
+	var/nofur = 0 		//Corgis that have risen past the material plane of existence.
```
Comment (ExcessiveUseOfCobblestone) [inline code/modules/mob/living/simple_animal/friendly/dog.dm]: 36/37 are boos, can you set them to T/F in all instances in this file?

**Review by ExcessiveUseOfCobblestone — CHANGES_REQUESTED** [review]

**Review by ninjanomnom — CHANGES_REQUESTED** [review]

File: code/modules/mob/living/simple_animal/friendly/dog.dm [hunk 2]
```diff
@@ -167,24 +185,31 @@
...
+					usr.put_in_hands(pcollar)
+					pcollar = null
+					update_corgi_fluff()
+					regenerate_icons()
 
 		show_inv(usr)
 
 	//Adding things to inventory
 	else if(href_list["add_inv"])
-		if(!Adjacent(usr) || !(ishuman(usr) || ismonkey(usr) || iscyborg(usr) ||  isalienadult(usr)))
```
[the remark this answers is not in the document]
Comment (ShizCalev) [reply code/modules/mob/living/simple_animal/friendly/dog.dm]: It's already handled by canUseTopic

**Pull request merged**